ethereum/permissioning/src/main/java/org/hyperledger/besu/ethereum/permissioning/node/provider/SyncStatusNodePermissioningProvider.java

```diff
@@ -91,7 +91,9 @@ private void handleInSyncEvent(final boolean isInSync) {
    *     otherwise
    */
   @Override
-  public boolean isPermitted(final EnodeURL sourceEnode, final EnodeURL destinationEnode) {
+  public boolean isConnectionPermitted(
+      final org.hyperledger.besu.plugin.data.EnodeURL sourceEnode,
```

Comment:
I think we really need to rename to avoid the package specification when using `EnodeURL` interface

Reply:
+1